cmd/aro/mirror.go

```diff
@@ -162,9 +176,6 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 		"registry.access.redhat.com/ubi7/go-toolset:1.16.12",
 		"registry.access.redhat.com/ubi8/go-toolset:1.17.7",
 		"mcr.microsoft.com/azure-cli:latest",
-
-		"quay.io/app-sre/managed-upgrade-operator:v0.1.856-eebbe07",
-		"quay.io/app-sre/hive:fec14dc",
```

Comment:
we use the hive commit as the tag in our [deployment of hive pipeline](https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/ARO.Pipelines?path=/ev2/generator/templates/hive-deployment/hive-deployment.tpl&version=GBmaster&line=277&lineEnd=278&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents). Will we need to update how we are deploying hive since we are changing how we are mirroring the image? From what I understand the answer should be 'no' as this new mirroring method should grab all tags from a date, and the tag should still remain as first 7 of the commit.